data/events.yml

```diff
@@ -0,0 +1,35 @@
+events:
+  - name: "🍷 Flux Kiwi Ruby Pre Party"
+    time: "Thursday 2 November, 5:30pm"
+    location: "<a href='https://www.thecross.co.nz/'>Southern Cross</a>, 39 Abel Smith Street"
+    payment: "Cash bar"
+    description_paragraphs:
+      - "Holding text for when our lovely friends at Flux give us some copy about this event."
+      - "It will be some kind of party at Southern Cross, where probably people hang out and talk."
+  - name: "🍻 Kiwi Ruby After Party"
+    time: "Friday 3 November, 5:30pm"
+    location: "Macs Brew Bar, Taranaki Street Wharf"
+    payment: "Snacks provided, tab for non-alcoholic drinks, cash bar for alcohol"
+    description_paragraphs:
+      - "We will have an after party for you to talk about the exciting things you learned and thought about at Kiwi Ruby."
+      - "Karaoke may ensue."
+  - name: "🐦🦆 Zealandia tour"
+    time: "Saturday 4 November, 12:45pm"
+    location: "Meet outside St James Theatre, Courtenay Place"
+    description_paragraphs:
+      - "New Zealand is home to a wonderful array of native birds, from the iconic flightless Kiwi that adorns much of our national imagery, to the cheeky kākāpō Sirocco, which you may know in your Slack channels as <a href='http://cultofthepartyparrot.com/'>:partyparrot:</a>."
+      - "If you’re a nature loving Rubyist, you may like to come on a trip to <a href='https://www.visitzealandia.com/'>Zealandia</a>, the world’s first fully-fenced urban ecosanctuary."
+      - "Bring warm clothes and your walking shoes. Zealandia is suitable for the whole family, and has a number of stroller and wheelchair accessible trails."
+    payment: "Bring $30 for admission and bus fare"
```

Comment:
@niceking how much actually is this?

Reply:
It's $19.50 for entry and the bus is $3.50 each way. I was going to gauge interest for tours on the sign up sheet cos it's cheaper in a group so I'd like the wording to be ambiguous. 

Can you change this copy to read 

Bring money for <a href='https://www.visitzealandia.com/Visit#VisitingHours'>admission</a> and $7 cash (or your Snapper) for return bus fare per adult.

Reply:
It's a pretty small space so I'm putting that in the main copy and just "Bring admission and bus fare" on the side

<img width="843" alt="screen shot 2017-09-06 at 6 35 23 pm" src="https://user-images.githubusercontent.com/1615322/30097524-2d75a040-9332-11e7-9e1b-b2a2d551826b.png">